ambry-api/src/main/java/com.github.ambry/config/RouterConfig.java

```diff
@@ -298,6 +298,35 @@
   @Default("1000")
   public final long routerOperationTrackerMinDataPointsRequired;
 
+  /**
+   * The maximum number of inflight requests that allowed for adaptive tracker. If current number of inflight requests
+   * is larger than or equal to this threshold, tracker shouldn't send out any request even though the oldest is past due.
```

Comment:
minor: Seems like all the other adaptive configs start with router.operation.tracker, even if they are only relevant to `AdaptiveOperationTracker`. Could we keep this the same?

Reply:
Could you also document the difference between parallelism and max.inflight.requests?

Reply:
sure, will do